src/lib.rs

```diff
@@ -44,28 +44,22 @@ pub mod note_bytes;
 
 use note_bytes::NoteBytes;
 
-/// The size of a compact note for Sapling and Orchard Vanilla.
+/// The size of a compact note for Sapling and pre-ZSA Orchard.
 pub const COMPACT_NOTE_SIZE: usize = 1 + // version
     11 + // diversifier
     8  + // value
     32; // rseed (or rcm prior to ZIP 212)
-/// The size of `NotePlaintextBytes` for Sapling and Orchard Vanilla.
+/// The size of [`Domain::NotePlaintextBytes`] for Sapling and pre-ZSA Orchard.
 pub const NOTE_PLAINTEXT_SIZE: usize = COMPACT_NOTE_SIZE + 512;
-
-/// The size of the memo.
-pub const MEMO_SIZE: usize = 512;
-/// The size of the authentication tag used for note encryption.
-pub const AEAD_TAG_SIZE: usize = 16;
-
 /// The size of [`OutPlaintextBytes`].
 pub const OUT_PLAINTEXT_SIZE: usize = 32 + // pk_d
     32; // esk
+const AEAD_TAG_SIZE: usize = 16;
+/// The size of an encrypted note plaintext for Sapling and pre-ZSA Orchard.
+pub const ENC_CIPHERTEXT_SIZE: usize = NOTE_PLAINTEXT_SIZE + AEAD_TAG_SIZE;
```

Comment:
remove `COMPACT_NOTE_SIZE`, `NOTE_PLAINTEXT_SIZE`, `ENC_CIPHERTEXT_SIZE` from this file.
These are domain specific values and should be part of the implementation.

Reply:
Done.